torstenstein-bio.md

```diff
@@ -0,0 +1,3 @@
+# Torsten Stein
+
+- Hanging out in Redmond
```

Comment:
Nice to have cooler weather in the Seattle area...  

Are you from the area originally?